tests/unittests/dataset_tests.py

```
@@ -4733,6 +4858,82 @@ def test_dataset_info_import_export(self):
                 dataset3.default_skeleton, dataset.default_skeleton
             )
 
+            self._assert_camera_intrinsics_equal(
+                dataset3.camera_intrinsics, dataset.camera_intrinsics
+            )
+            self._assert_sensor_extrinsics_equal(
+                dataset3.sensor_extrinsics, dataset.sensor_extrinsics
+            )
+
+    def _assert_camera_intrinsics_equal(self, actual, expected):
```

Ruff (0.14.10)

Missing return type annotation for private function `_assert_camera_intrinsics_equal`

Add return type annotation: `None`

(ANN202)